ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/flow_trackings.schema.yml

```diff
@@ -0,0 +1,87 @@
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  flow_trackings:
+    type: list
+    primary_key: type
+    items:
+      type: dict
+      keys:
+        type:
+          type: str
+          valid_values: ['sampled']
+          description: Flow Tracking Type - only 'sampled' supported for now
+        sample:
+          type: int
+          convert_types:
+            - str
+        trackers:
+          type: list
+          primary_key: name
+          convert_types:
```

Comment:
since this is a new model, we can skip the convert_types from dicts. We will only use the new data models.

Reply:
Fixed in e00f2f0